lib/main.dart

```diff
@@ -46,12 +54,14 @@ class _MyHomePageState extends State<MyHomePage> {
       });
     } else {
       setState(() {
+        final chatMessage = ChatMessage(
+          time: DateTime.now().millisecondsSinceEpoch,
+          message: textMessageController.text,
+        );
+        FirebaseFirestore.instance.collection('chat').add(chatMessage.toJson());
         messages.insert(
```

Comment:
I just thought now, probably we want to implement stream first.
This way, we can show we can manually create the data in the Firebase. what do you think?

Also, you don't need this duplicate logic even temporarily. Call me if you need more explanation.

Reply:
I don't think so. If we do have time, we can do it after the stream step, no?

Reply:
Let's talk about it during sync-up in 30min

Reply:
Let's go now.